poc_v2.1/app.py

```
@@ -43,11 +52,11 @@ def ask_star(user_query, role):
                 label="Specify your role (optional):",
-                value="general"  # Default role
+                value="general",
             ),
         ],
         outputs=gr.Textbox(label="Star's Response:"),
         title="Star - Massachusetts Tenant Law Expert",
         description="Ask Star any question you have about tenant law in Massachusetts and she will provide an expert answer based on the provided documents.",
     )
-    iface.launch(server_name="0.0.0.0", server_port=7860) # Make it accessible on your network
\ No newline at end of file
+    iface.launch(server_name="0.0.0.0", server_port=7860)
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)